lib/suspenders/generators/stylelint_generator.rb

```diff
@@ -0,0 +1,22 @@
+require_relative "base"
+
+module Suspenders
+  class StylelintGenerator < Generators::Base
+    def create_stylelintrc
+      copy_file "stylelintrc.json", ".stylelintrc.json"
+    end
+
+    def install_dependencies
+      case self.behavior
+      when :invoke
+        run "bin/yarn add stylelint --dev"
+        run "bin/yarn add @thoughtbot/stylelint-config --dev"
+      when :revoke
+        self.behavior = :invoke
```

Comment:
This seems silly, but I don't know if there is a better way. The run command doesn't do anything when the behavior is `:revoke`: https://github.com/erikhuda/thor/blob/d3762d0b89a80da488285d640b052f5276fac6bb/lib/thor/actions.rb#L245-L246

Reply:
I had to do something similar for the hound commenting. Maybe I need some sort of custom action here?